src/commit_rules.rs

```diff
@@ -102,40 +98,44 @@ const WHAT_AND_WHY_EXPLANATION: CommitRule = CommitRule {
     text: "Use the body to explain what and why vs. how",
 };
 
-trait FetchedAdd {
-    fn fetch_add (&mut self, value: Self) -> Self;
+fn test_starting_emoji (input: &str) -> bool {
+    CommitType::iter_variants().fold(false, |valid, commit_type| {
+        valid || input.starts_with(commit_type.emoji())
+    })
```

Comment:
Could this use [`any`](https://doc.rust-lang.org/rust-by-example/fn/closures/closure_examples/iter_any.html) instead?

```suggestion
    CommitType::iter_variants().any(|commit_type| {
        input.starts_with(commit_type.emoji())
    })
```